src/Tribe/Attendees.php

```diff
@@ -820,6 +825,47 @@ public function maybe_generate_csv() {
 		}
 	}
 
+	/**
+	 * Admin requested a CSV export from the attendees table.
+	 * Generates the download and finishes the execution.
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function export_csv_for_all() {
+		if ( empty( $_GET['attendees_csv'] ) || empty( $_GET['attendees_csv_nonce'] ) || ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
```

Comment:
I would love to see this being separated in to three separate conditionals with a bail for each. Helps readability.